Source/Resources/Content/MSGraph/DeviceManagement/DeviceManagementScripts/Management - Set-LocalAdminPassword.json

```diff
@@ -7,7 +7,7 @@
       }
     }
   ],
-  "description": "Creates or updates a local computer account with administrator rights, and sets portions of the last used BitLocker recovery key as its password.",
+  "description": "What is it: Creates or updates a local computer account with administrator rights, and sets portions of the last used BitLocker recovery key as its password. ",
```

Comment:
**What does this do?**
Creates a local administrator account on Windows devices. This account password is set from the first three portions of the device's BitLocker recovery key (which can be viewed in the Azure portal) plus the letter "X" (e.g. 123456-123456-123456X)

**Why should you use this?**
IT Support personnel with permission to view the BitLocker recovery key will be able to log in to Windows devices using a local administrator account.

**What is the end-user impact?**
None. 

**Learn more**
[View BitLocker recovery keys](https://365adviser.com/azure/how-to-find-the-bitlocker-recovery-key-in-azure-ad)